hathor/p2p/utils.py

```diff
@@ -20,8 +20,8 @@
 import requests
 from cryptography import x509
 from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.backends.openssl.rsa import RSAPrivateKey
 from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey
```

Comment:
Are you sure this is still using OpenSSL (instead of a custom implementation from the package)?

Reply:
I removed this from this PR.

Reply:
Yes, OpenSSL is just the backend. In fact we were already importing the same `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey`, as a re-export in `cryptography.hazmat.backends.openssl.rsa`. Check it out in the source code for our current version (`38.0.3`):

https://github.com/pyca/cryptography/blob/38.0.3/src/cryptography/hazmat/backends/openssl/rsa.py#L31-L32

And in the source code for the new version (`42.0.5`), check that `RSAPrivateKey` is indeed being registered with an OpenSSL backend:

https://github.com/pyca/cryptography/blob/42.0.5/src/cryptography/hazmat/primitives/asymmetric/rsa.py#L67

Reply:
Great! Thanks!